### DISCUSSION TITLE
Wrong icon?

### DISCUSSION THREAD

**Discussion by kelenorbi** [containrrr/watchtower#1371] [General] [answered]
I wonder why you use a lighthouse icon instead of a watchtower?

https://wikidiff.com/lighthouse/watchtower

**Comment by piksel** [comment, marked as answer]
I wanted to distance it from the connotations (prisons, military etc.). The old logotype depicted something closer to the generic definition, but I figured this better represented the role of watchtower. It also aligned it with the sea theme of containers. 

It was @simskij who drew the logo though. I were just the one complaining :)

**Reply by simonaronsson** [reply]
Yeah, the original logo was a watchtower, with a gopher standing guard, but we decided against that as the connotations didn't feel excellent.

![image](https://user-images.githubusercontent.com/1596025/192273821-aebd5f88-e91c-4be4-900e-384ae9cb55fc.png)
